src/custom.rs

```diff
@@ -45,17 +48,24 @@ pub enum ConfigLoadError {
     Invalid(#[from] serde_yaml::Error),
 }
 
-#[derive(Clone, Debug, Deserialize)]
+// TODO(JsonSchema): suggest the valid themes based on themes.presentation.theme_names() but allow
```

Comment:
As I commented before, I think this isn't necessary (this may be a leftover comment!)

Reply:
Yep 😄 
fixed now.